content/en/deployment/strategies/blue-green.md

```diff
@@ -88,3 +106,50 @@ Monitor the progress and **Approve & Continue** or **Roll back** the blue/green
 
 
 
+## Optional configuration
+
+### Preview service
+You can also create a `previewService` Kubernetes Service object so you can programmatically or manually observe the new version of your software before exposing it to traffic via the `activeService`. This is the `trafficManagement.kubernetes.previewService` field in the YAML configuration.
+
+In order to have a separate preview service serving traffic to only your new version, make sure to deploy your new version under a new app name.
```

Comment:
This line is adding more confusion, partly because it says to deploy something.

I'd suggest removing it or re-wording it.